### PR TITLE
feat: allow specifying expected images directory

### PR DESCRIPTION
**Pull request by kopancek**
This way, it is not needed to find previous runs and download an artifact. Instead, the images are copied from the expected images directory and compared immediately.

### CODE REVIEW

**Comment by kopancek** [comment]
@bokuweb friendly ping, any chance you might have capacity to look at this PR? 🙏

**Comment by bokuweb** [comment]
@kopancek thanks!i'l l check

File: src/comment.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export type CreateCommentWithTargetInput = {
   sha: string;
   regBranch: string;
   artifactName: string;
-  targetRun: Run;
+  targetHash: string | null;
```
Comment (bokuweb) [inline src/comment.ts:13]: Why did you edit this part? This change is unnecessary for this feature, and I would appreciate it if you could keep changes to a minimum.

File: src/config.ts [hunk 2]
```diff
@@ -114,6 +118,8 @@ export const getConfig = (): Config => {
   const imageDirectoryPath = core.getInput('image-directory-path');
   validateGitHubToken(githubToken);
   validateImageDirPath(imageDirectoryPath);
+  const expectedImagesDirectoryPath = core.getInput('expected-images-directory-path') ?? null;
+  validateImageDirPath(imageDirectoryPath, true);
```
Comment (bokuweb) [inline src/config.ts:122]: maybe it is not needed to add `allowEmpty` flag.
```suggestion
if (expectedImagesDirectoryPath) {
  validateImageDirPath(expectedImagesDirectoryPath);
}
```

File: src/comment.ts (hunk 1 above)
Comment (kopancek) [reply src/comment.ts]: Reverted, marking the comment as resolved 👍

File: src/config.ts (hunk 2 above)
Comment (kopancek) [reply src/config.ts]: Sorry, bad code from me. Changed accordingly, but using a `null` check instead.

**Comment by kopancek** [comment]
@bokuweb I think I addressed all of your comments. Would you be able to take another look when time allows? Thanks :)

File: src/config.ts [hunk 3]
```diff
@@ -27,7 +28,10 @@ const validateGitHubToken = (githubToken: string | undefined) => {
   }
 };
 
-const validateImageDirPath = (path: string | undefined) => {
+const validateImageDirPath = (path: string | undefined, allowEmpty: boolean = false) => {
```
Comment (bokuweb) [inline src/config.ts:31]: please remove `allowEmpty`

File: src/config.ts [hunk 4]
```diff
@@ -27,7 +28,10 @@ const validateGitHubToken = (githubToken: string | undefined) => {
...
+  if (allowEmpty && !path) {
+    return;
+  }
```
Comment (kopancek) [inline src/config.ts:34]:
```suggestion
const validateImageDirPath = (path: string | undefined) => {
```

File: src/config.ts (hunk 3 above)
Comment (kopancek) [reply src/config.ts]: Updated 👍

**Comment by bokuweb** [comment]
LGTM 

I'll merge into develop branch for now.

**Pull request merged**